docs/README.es.md

```diff
@@ -0,0 +1,30 @@
+<h3 align="center"><img src="../assets/onefetch.svg" height="130px"></h3>
+
+<h5 align="center">Una herramienta de información de Git de línea de comandos escrita en Rust</h5>
+
+<p align="center">
+	<a href="https://crates.io/crates/onefetch"><img src="https://img.shields.io/crates/v/onefetch.svg" alt="cargo"></a>
+	<a href="https://github.com/o2sh/onefetch/actions"><img src="https://github.com/o2sh/onefetch/workflows/CI/badge.svg" alt="Build Status"></a>
+  <a href="https://github.com/o2sh/onefetch/wiki/language-Support"><img src="../assets/language-badge.svg"></a>
+	<a href="https://github.com/o2sh/onefetch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22"><img src="https://img.shields.io/github/issues/o2sh/onefetch/help%20wanted?color=green" alt="help wanted"></a>
+	<a href="./LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
+	<img src="../assets/msrv-badge.svg">
+</p>
+
+<img src="../assets/screenshot-1.png" align="right" height="240px">
+
+Onefetch es una herramienta de información Git de línea de comandos escrita en `Rust` que muestra información del proyecto y estadísticas de código para un repositorio Git local directamente en su terminal. La herramienta funciona sin conexión a internet.
+
+De manera predeterminada, la información del repositorio se muestra junto con el logotipo del idioma dominante, pero puede configurar onefetch para usar una imagen, en terminales compatibles, una entrada de texto o nada en absoluto.
```

Comment:
I'd change "la información del repositorio se muestra junto con el logotipo del idioma dominante" to "la información del repositorio se muestra junto con el logotipo del lenguaje dominante"

we use "idioma" if we talk about a spoken language (English, Japanese, French, etc)